packages/ord-connect/src/assets/phantom-wallet.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="1200" height="1200" viewBox="0 0 1200 1200" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg width="24" height="24" viewBox="0 0 1200 1200" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Just changing the number here so that intrinsically it doesn't render such a large size